pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1835,6 +1835,21 @@ class Timedelta(_Timedelta):
         Values for construction in compat with datetime.timedelta.
         Numpy ints and floats will be coerced to python ints and floats.
 
+    **Attributes
```

Comment:
this is probably fine, but you need to remove the leading `**`

Reply:
Thanks for the review. I've removed the leading `**`.